libs/librepcb/editor/dialogs/aboutdialog.cpp

```diff
@@ -146,6 +145,30 @@ AboutDialog::AboutDialog(const WorkspaceSettings& settings,
     mUi->lblContributing->setText(text);
   }
 
+  // Set credits text (not using Qt designer to avoid layouting issues).
+  {
+    QString text;
+    text += "<p>" %
+        tr("This project relies on many contributors, sponsors and other "
```

Comment:
You could link "contributors" to https://github.com/LibrePCB/LibrePCB/graphs/contributors 🙂

Reply:
I didn't link to this site because this link already exists on the first tab and I don't like a redundant link:

![image](https://github.com/LibrePCB/LibrePCB/assets/5374821/89046cd9-1c0a-49e5-868f-c61a399d34f4)

Not sure if this information should be moved to the "Credits" tab as well, but then I don't know what to write on the first tab :see_no_evil: If you have any suggestion, let me know :)

Reply:
Ah, true. I don't think it would hurt to link to it twice though.

Reply:
Okay, added the link :+1: